Makefile

```diff
@@ -0,0 +1,33 @@
+DOCKER_IMAGE = oam/server:latest
+
+all: server
+
+server:
+	@docker build -f ./server/Dockerfile -t $(DOCKER_IMAGE) ./server
```

Comment:
Documenting the minimum version is a good idea.

Just to give you an idea of how I worked on this, I use [`docker-machine`](https://docs.docker.com/machine/):

```bash
$ # Only the first time
$ docker-machine create -d virtualbox dev
$ docker-machine start dev && docker upgrade dev
$ # Do work
$ docker-machine stop dev
```

One missing step is setting up port forwarding for the VirtualBox host-only interface. You don't really have to do that if you use the host-only IP address to interact with the service running inside the container vs. `localhost`.